examples/acm_data.py

```diff
@@ -80,11 +87,14 @@ def create_acm_graph(dowload_path='/tmp/ACM.mat', output_path=None):
     
     parser.add_argument('--download-path', type=str, default='/tmp/ACM.mat',
                         help="The path of folder to store downloaded ACM raw data")
+    parser.add_argument('--dataset-name', type=str, default='acm',
```

Comment:
Are you going to support other datasets?

Reply:
The name of the example is already acm_data.py I don't think we need this argument.

Reply:
I provide this argument to let users know that they can change the name.

Reply:
If they change the name, it is still acm, right?

Reply:
yes, the data is still acm data, but the saved file will be <theirname>.dgl. And in the rest steps they need to use their name to replace "acm" in all scripts.